alt_eval.py

```diff
@@ -22,7 +22,7 @@ def _eval(path_to_checkpoint_file, path_to_data_dir, path_to_log_dir, lmdb_file,
     print(f'Evaluate {path_to_checkpoint_file} on {path_to_eval_lmdb_dir}')
     results = AltEvaluator(path_to_eval_lmdb_dir, number_of_images_to_evaluate).evaluate(model)
 
-    export_evaluate_to_data_dir(path_to_data_dir, results, get_model_version(path_to_checkpoint_file))
+    export_evaluate_to_data_dir(path_to_log_dir, results, get_model_version(path_to_checkpoint_file))
```

Comment:
The reason I'm storing the output in the datadir is so that I can access the json from model academy (we can change this once we've gotten rid of submodule.run(s) and pass the evaluate data through json) but for now I think just let the results be stored in the data dir instead of the log dir

tldr: we don't need this commit